src/tfc_api.ts

```diff
@@ -23,11 +25,27 @@ export class TerraformCloudApi {
         this.tfcApiToken = tfcApiToken;
         this.orgId = orgId;
         this.workspaceName = workspaceName;
+        this.workspaceId = "";
         // for mocking
         this.__fetch = fetchMock ? fetchMock : fetch;
     }
 
-    public async setVariable(workspaceId, existingValue, name, value): Promise<boolean> {
+    public configBlock(): string {
+        return (
+`terraform {
+    cloud {
+        hostname     = "${this.baseDomain}"
+        organization = "${this.orgId}"
+        workspaces {
+        name = "${this.workspaceName}"
+        }
+    }
+}`
+        );
+    }
```

Comment:
this used to happen within TerraformCli#tfInit(), but was moved out here to make the config swappable